package ex08.example2;

public class TryEx02 {
    public static void main(String[] args) {
        throw new RuntimeException("오류 강제 발생");
        // 매서드 내에서 예외상황을 명시적으로 알리는 역할
    }
}
